### PR TITLE
EOS-24017: Added SSPL directory at the end of the given path for SB generation.

### PR DESCRIPTION
**Pull request by satish-seagate**
Signed-off-by: Satish Darade <satish.darade@seagate.com>

## Coding:

* [ ] Did you follow coding standard and verified it with flake8? 

* [ ] Did your address all Codacy issues?

## Testing:

* [ ] Unittest updated and all unittests are passing?

* [ ] Sanity/Self tests are updated (if applicable)?

* [ ] Manual testing done covering happy path and non-happy path?

## Integration:

* [ ] If there is any interface change, did you communicate it to other components?
* [ ] If there is any interface change, other componenent gate gatekeepers are ready to accept the change?
* [ ] Did you complete deployment test if applicable?

## PR checklist:
* [ ] Did you add Jira number to the PR?

* [ ] DCO and cla-signed?

### CODE REVIEW

**Review by cdeshmukh — APPROVED** [review]

File: low-level/solution/lr2/support_bundle/sspl_support_bundle [hunk 1]
```diff
@@ -195,7 +195,7 @@ class SupportBundle(object):
 
         if file_path:
             if os.path.exists(file_path):
-                self.tmp_bundle_path = os.path.normpath(file_path)
+                self.tmp_bundle_path = os.path.normpath(file_path+"sspl")
```
Comment (jjxsg) [inline low-level/solution/lr2/support_bundle/sspl_support_bundle:198]: Unless we use default path, **file_path** will be coming from user who may not add '/' at the end of the path. So, if I give path `/var/cortx` for sspl support bundle the resultant path will become `/var/cortxsspl`
Add `/sspl`, `normpath` will take care of any extra '/' in the path.

Comment (satish-seagate) [reply low-level/solution/lr2/support_bundle/sspl_support_bundle:198]: I don't feel it is required.
I have tested it in VM as well as the hw setup and it is working as expected.

**Review by vastradparayya — APPROVED** [review]

File: low-level/solution/lr2/support_bundle/sspl_support_bundle (hunk 1 above)
Comment (jjxsg) [reply low-level/solution/lr2/support_bundle/sspl_support_bundle:198]: Can you please try below command and see if you are getting correct results:
`/opt/seagate/cortx/sspl/bin/sspl_support_bundle create test_bundle /tmp/sat001 -console`

Comment (satish-seagate) [reply low-level/solution/lr2/support_bundle/sspl_support_bundle]: Yes @jjxsg, for the above case it is appending an SSPL at the end. made required chnages.
Done.

**Review by jjxsg — APPROVED** [review]

**Pull request merged**